react-app/src/view/Users/User.jsx

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import axiosInstance from "../../axios";
+import { base } from "../../api";
+import {
+  Container,
+  Grid,
+  Paper,
+  Typography,
+  Button,
+  Stack,
+  InputLabel,
+  Select,
+  MenuItem,
+} from "@mui/material";
+import { useNavigate } from "react-router-dom";
+
+const User = () => {
+  const { id } = useParams();
+  const [singleUser, setSingleUser] = useState([]);
+  const [roleList, setRoleList] = useState([]);
+  const [choosedRole, setChoosedRole] = useState(3);
+  const navigate = useNavigate();
+
+  function handleChange(event) {
+    // handle change for select input (role)
+    console.log(event.target.value);
+    setChoosedRole(event.target.value);
+  }
+
+  function handleFunc() {
+    deleteUser();
+    navigate("/users");
+  }
```

Comment:
`await` `deleteUser` before navigating. And this function seems to not be doing a lot. In that case, you can just move over `navigate("/users");` into `deleteUser` function.